### PR TITLE
Adding PowerShell & Graph permission information as well as link to Graph documentation

### PR DESCRIPTION
**Pull request by MSFTRickyCastaneda**
Added a section outlining what permissions to use when connecting tenant to graph using PowerShell as well as a link to the Graph PowerShell documentation. Please review and leave comments for review.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/408be8168a4ab55552139bdbb98e359c42b08159/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 94 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/405dbe9c-ed46-47f0-84be-10aa9fb5412c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/74d90144be9c2299db61874f14e896380b651bfc/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 94 | 25 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ca08f09e-1677-4871-9eb4-36c70a408b74) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by subray2014 — APPROVED** [review]

File: msteams-platform/graph-api/rsc/grant-resource-specific-consent.md [hunk 1]
```diff
@@ -425,7 +425,15 @@ The default value of the property `isUserPersonalScopeResourceSpecificConsentEna
 
 <b> After March 14th, 2024 </b>
 
-Admins adjust team and chat RSC settings through PowerShell cmdlets. The following are the available states for the PowerShell settings and each section shows examples of how to use these states to adjust your settings:
+Before you can manage your Team and Chat RSC settings with PowerShell, you need to connect PowerShell to your tenant using Microsoft Graph. You can learn more about managing Microsoft Graph settings with PowerShell [here](/powershell/microsoftgraph/get-started?view=graph-powershell-1.0).
```
Comment (v-sdhakshina) [inline msteams-platform/graph-api/rsc/grant-resource-specific-consent.md:428]:
```suggestion
Before you manage your team and chat RSC settings with PowerShell, you need to connect PowerShell to your tenant using Microsoft Graph. For more information on managing Microsoft Graph settings with PowerShell, see [get started with the Microsoft Graph PowerShell SDK](/powershell/microsoftgraph/get-started?view=graph-powershell-1.0).
```

File: msteams-platform/graph-api/rsc/grant-resource-specific-consent.md [hunk 2]
```diff
@@ -425,7 +425,15 @@ The default value of the property `isUserPersonalScopeResourceSpecificConsentEna
...
+You can use the `Connect-MgGraph` cmdlet to do this. You will need to connect with the following permissions.
```
Comment (v-sdhakshina) [inline msteams-platform/graph-api/rsc/grant-resource-specific-consent.md:429]:
```suggestion
You can use the `Connect-MgGraph` cmdlet and connect with the following permissions:
```

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bf8a6fee5c7a8ef688d8a043c9337b56557ba711/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a1a35d0-6534-49c2-8955-0040b99488d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d593aada3b13ad9dc37911df172431bb0d8fbfdb/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/96bc2e38-9df9-40df-be32-a80ce3ccf1ee) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3ee1af2d798ffbb6c4393004ad9bfdd905d70727/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/21c82865-e05e-4942-9459-e60fa53d4bc3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/53b1b671ebdb8a330bae2df1390be73320ec876d/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d566ce5b-8206-4995-aac5-863265dd3c2b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2ab87505c781f0d00e9f1c6f9fc5d922c8c52e76/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5bf7a3ce-f924-41cd-94fa-ebb91b93726d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/graph-api/rsc/grant-resource-specific-consent.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a5f23ecb40806ee7d8d39f0282699118fa2f3873/msteams-platform%2Fgraph-api%2Frsc%2Fgrant-resource-specific-consent.md) | 95 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/85b3a061-24e6-4ce4-95d3-c76632300dc0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**